Tella/Scenes/CommonConnectionReport/Views/Draft/AddFilesToDraftView.swift

```diff
@@ -1,15 +1,18 @@
+//
+//  AddFilesToDraftView.swift
 //  Tella
 //
-//  Copyright © 2022 INTERNEWS. All rights reserved.
+//  Created by gus valbuena on 6/24/24.
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
 //
 
 import SwiftUI
 
-struct AddFilesToDraftView: View {
+struct AddFilesToDraftView<T: ServerProtocol>: View {
     
     @EnvironmentObject var appModel: MainAppModel
     @EnvironmentObject var sheetManager: SheetManager
-    @EnvironmentObject var draftReportVM: DraftReportVM
+    @EnvironmentObject var draftReportVM: DraftMainViewModel<T>
     
```

Comment:
this should be @StateObject and not @EnvironmentObject and then passed in parameter when it is needed